### PR TITLE
[PAY-779][PAY-781] Add premium track page and tile on mobile - Part 3

### PR DESCRIPTION
**Pull request by sddioulde**
### Description

Part 3 of the work that adds premium track page and tile to mobile.
Will merge into https://github.com/AudiusProject/audius-client/pull/2819.

### Dragons

n/a

### How Has This Been Tested?

local ios dapp against stage

### How will this change be monitored?

n/a

### Feature Flags ###

existing premium content feature flags

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-pc-mobile-track-page-tile-3

File: packages/mobile/src/components/lineup-tile/LineupTileTopRight.tsx [hunk 1]
```diff
@@ -77,25 +85,52 @@ type Props = {
...
+  const { neutralLight4 } = useThemeColors()
+  const accentBlue = useColor('accentBlue')
   const trackTileStyles = useTrackTileStyles()
 
   return (
     <View style={styles.topRight}>
-      {showArtistPick && isArtistPick && (
-        <LineupTileTopRightItem icon={IconStar} label={messages.artistPick} />
+      {!!premiumConditions && (
+        <LineupTileTopRightItem
```
Comment (dharit-tan) [inline packages/mobile/src/components/lineup-tile/LineupTileTopRight.tsx:108]: It's top left on web but top right on mobile? Haven't checked figma but ok.

Comment (sddioulde) [reply packages/mobile/src/components/lineup-tile/LineupTileTopRight.tsx:108]: this is different than the corner tag. it's a label. web also has it on top right

File: packages/mobile/src/components/lineup-tile/LineupTileAccessStatus.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    fontFamily: typography.fontByWeight.bold,
+    fontSize: typography.fontSize.medium,
+    color: palette.neutralLight4
+  }
+}))
+
+export const LineupTileAccessStatus = ({ trackId }: { trackId: ID }) => {
+  const styles = useStyles()
+  const premiumTrackStatusMap = useSelector(getPremiumTrackStatusMap)
+  const premiumTrackStatus = premiumTrackStatusMap[trackId]
```
Comment (piazzatron) [inline packages/mobile/src/components/lineup-tile/LineupTileAccessStatus.tsx:41]: maybe worthwhile to have a selector that does both of these steps?

**Review by piazzatron — APPROVED** [review]

File: packages/mobile/src/components/lineup-tile/LineupTileAccessStatus.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  const neutralLight4 = useColor('neutralLight4')
```
[the remark this answers is not in the document]
Comment (sddioulde) [reply packages/mobile/src/components/lineup-tile/LineupTileAccessStatus.tsx:42]: useColor uses theme colors

**Pull request merged**